src/components/NewMessageForm/NewMessageForm.vue

```diff
@@ -387,12 +369,6 @@ export default {
 
 			range.setStartAfter(emojiTextNode)
 		},
-
-		// Check whether the current conversation is the first in the conversations
-		// list and stores the value in the component's data.
-		isCurrentConversationIsFirstInList() {
-			this.conversationIsFirstInList = this.$store.getters.conversationsList.map(conversation => conversation.token).indexOf(this.token) === 0
```

Comment:
This didn't work when you had a favorite and posted in a normal conversation?

Reply:
I just tested in this PR and having a favorite or normal conversation makes no difference.
It just makes sure that whatever room you posted in becomes visible in the left panel.

And when posting in a room that is invisible below the scroll panel, the room gets moved to the top, below favorites, highlighted, and it scrolls to that.

@ma12-co do you know why there was a check for "first room" ?

Reply:
Don't remember honestly :/